webcomponents/tool/src/main/frontend/js/sui-notifications/sui-notifications.js

```diff
@@ -110,6 +116,24 @@ class SuiNotifications extends SakaiElement {
     }
   }
 
+  _decorateAssignmentNotification(noti) {
+    if (noti.event === "asn.new.assignment" || noti.event === "asn.revise.access") {
```

Comment:
I like to use a blank line to start the function body, unless it is a one line function.

```suggestion

    if (noti.event === "asn.new.assignment" || noti.event === "asn.revise.access") {
```